src/executors/killWithChildren.ts

```diff
@@ -0,0 +1,15 @@
+import { exec } from "child_process"
+
+export default (pid: number): Promise<void> => {
+    
+    return new Promise((resolve, reject) => {
+        if(process.platform === "win32") {
+            exec(`taskkill /pid ${pid} /T /F`, () => resolve());

```

Comment:
Could we use `execSync` here and make this method synchronous? This would fix the async promise callback issue from `ReplExecutor.ts`.

Reply:
Done!